### ISSUE TITLE
iptable_filter.ko not available, but iptables works

### ISSUE THREAD

**Issue by vadorovsky** [cilium/cilium#7892]
Reported by user on Slack:

```
level=fatal msg="iptables modules could not be initialized" error="could not load module ip_tables: exit status 1" subsys=iptables
```

That error occured on CoreOS Container Linux with kernel `4.19.34-coreos`. It has `iptable_filter.ko` module available, but not `ip_tables.ko`. And iptables still works.

**Comment by vadorovsky** [comment]
These are modules available on that system:

```
arp_tables.ko
arpt_mangle.ko
arptable_filter.ko
ipt_CLUSTERIP.ko
ipt_ECN.ko
ipt_MASQUERADE.ko
ipt_REJECT.ko
ipt_SYNPROXY.ko
ipt_ah.ko
ipt_rpfilter.ko
iptable_filter.ko
iptable_mangle.ko
iptable_nat.ko
iptable_raw.ko
iptable_security.ko
nf_defrag_ipv4.ko
nf_dup_ipv4.ko
nf_log_arp.ko
nf_log_ipv4.ko
nf_nat_h323.ko
nf_nat_ipv4.ko
nf_nat_pptp.ko
nf_nat_proto_gre.ko
nf_nat_snmp_basic.ko
nf_reject_ipv4.ko
nf_socket_ipv4.ko
nf_tproxy_ipv4.ko
nft_chain_nat_ipv4.ko
nft_chain_route_ipv4.ko
nft_dup_ipv4.ko
nft_fib_ipv4.ko
nft_masq_ipv4.ko
nft_redir_ipv4.ko
nft_reject_ipv4.ko
```

**Comment by CH-anhngo** [comment]
thank you for quick fix!

**Comment by brb** [comment]
Fixed by #7893.
